packages/core/botpress/CHANGELOG.md

```diff
@@ -31,7 +31,7 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 
 <a name="10.24.4"></a>
-## [10.24.4](https://github.com/botpress/botpress/compare/v10.24.3...v10.24.4) (2018-07-20)
+## 10.24.4 (2018-07-20)
```

Comment:
In some entries, dates are gone but here it's left.

Reply:
Because we deprecated `10.24.3` but not `10.23.4` by this reason this link is broken.